docs/how_to_guide.rst

```diff
@@ -28,19 +69,34 @@ Access
     In both the implicit and explicit access, role data has a context that tells which resource(s)
     the requesting user can access
 
-
 Implementation
 --------------
 To add RBAC implicit and explicit authorization checks, you need to follow the below steps. Here we are using ``LMS``,
 `edx-enterprise-data <https://github.com/edx/edx-enterprise-data/>`_ and `ecommece <https://github.com/edx/ecommerce>`_
-codebases as an example.
+codebases as an example.  There is also an `RBAC demo repo <https://github.com/iloveagent57/edx_rbac_demo/>`_ that
+demonstrates how to configure django-rules, set up feature roles/mappings, define rule predicates and permissions, and
+make use of those in a DRF viewset.
 
-1. In LMS create a `system wide role data migration <https://github.com/edx/edx-enterprise/blob/master/enterprise/migrations/0066_add_system_wide_enterprise_operator_role.py>`_. You only need to do this if you are creating a new role. We create
-system wide role when we want to give access to users to a system wide resource, for example, being a Course Instructor
-for a specific Course.
+1. `Configure Django <https://github.com/dfunckt/django-rules#configuring-django>`_ to use the ``rules`` package:
 
+.. code-block:: python
 
-2. For implicit access, In LMS create a system wide role assignment for a user using django admin at
+    INSTALLED_APPS = (
+        # ...
+        'rules.apps.AutodiscoverRulesConfig',
+    )
+    # ...
+    AUTHENTICATION_BACKENDS = (
+        # ...
+        'rules.permissions.ObjectPermissionBackend',
+        'django.contrib.auth.backends.ModelBackend',
+    )
+
+2. In LMS create a `system wide role data migration <https://github.com/edx/edx-enterprise/blob/master/enterprise/migrations/0066_add_system_wide_enterprise_operator_role.py>`_. You only need to do this if you are creating a new role. We create
+   system wide role when we want to give access to users to a system wide resource, for example, being a Course Instructor
+   for a specific Course.
+
+3. For implicit access, In LMS create a system wide role assignment for a user using django admin at
    ``/admin/enterprise/systemwideenterpriseuserroleassignment/``. LMS must have a ``SYSTEM_WIDE_ROLE_CLASSES`` django
    setting which contains the name of system wide role assignment model class. This will be used to add role data in JWT
    by LMS using ``create_role_auth_claim_for_user`` rbac util function. A ``SYSTEM_WIDE_ROLE_CLASSES`` django setting
```

Comment:
nit/proposed text: "This will be used by the LMS to add role data to the JWT by using the ... function"